packages/docs/themes/cerebral-website/menu-item/_theme/menu-item_theme_islands.css

```diff
@@ -1,5 +1,18 @@
+.menu-item_theme_islands {
+  padding: 0 20px !important;
```

Comment:
should avoid !important.
let's use the same selector for padding rule as defined in islands theme:
```css
.menu_size_xl .menu__group-title~.menu-item_theme_islands, .menu_size_xl[class*=menu_mode] .menu-item_theme_islands {
    padding: 0 20px;
}
```